journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
-and the jailer might see it if he pleased; and after I had read it to her,
-she was tenderly affected, turned to her brethren again, and said,
-surely these people will never come to us, but we must go to them.
+and the jailer could see it if he pleased. And after I had read it to her,
+she was tenderly affected, and turning to her brethren, she said,
+"`Surely these people will never come to us, but we must go to them.`"
 Some time after, through much trouble and affliction, she came to receive the truth,
-to live in it, and obey it; her name was Margaret Bowen, wife of John Bowen of Colfryn,
-and mother to Peter Bowen, in Martin`'s-le-Grand, London.
+and to live in it and obey it. Her name was Margaret Bowen,
+wife of John Bowen of Colfryn, and mother to Peter Bowen,
+in Martin`'s-le-Grand, London.
 
-In a little time my service was over among those professors in that prison,
-and the Lord made way for my enlargement.
-In about two weeks I came away, and left the rest of the prisoners there,
+In a little time, my service was over among those professors in that prison,
+and the Lord made way for my freedom.
+In about two weeks I was set at liberty, and left the rest of the prisoners there,
 where they continued a considerable time.
-I was well satisfied with the goodness of God, that I found his presence, life,
-and power with me; a present help in the time of need, which kept me low and humble,
-that I dared not rejoice that spirits were made subject; but rather rejoice,
+I was well satisfied with the goodness of God, in that I found His presence, life,
+and power with me--a present help in the time of need--which kept me low and humble,
+so that I dared not rejoice in those still under bondage, but rather rejoice
 that I found my name written in heaven.
 I came home in great love and peace to my wife and family,
 and many of my loving neighbours rejoiced to see me.
 
-I continued about my calling and business,
-and waited upon God to know his mind and will concerning me.
-In this time I heard of an honest old woman,
-who had received the truth some time before at Montgomery, her name was Ann Hamon,
-wife of Thomas Hamon; my wife and I went to visit her.
+After this I continued about my calling and business,
+and waited upon God to know His mind and will concerning me.
+I heard of an honest old woman
+who had received the truth some time before at Montgomery. Her name was Ann Hamon,
+wife of Thomas Hamon; and my wife and I went to visit her.
 
-About the year 1661, I went to a meeting at Edgemont near Wem, in Shropshire.
+About the year 1661, I went to a meeting at Edgemont, near Wem, in Shropshire.
 Here our friend William Gibson and I were taken prisoners,
 with about twenty-five or twenty-six more, and sent to Shrewsbury.
 We found the temper of the jailer to be very cruel.
-He threatened us with a great deal of hardship, if we did not eat of his food,
+He threatened us with a great deal of hardship if we did not eat of his food,
```

Comment:
Let's tweak this to say: if we did not eat his food, drink his drink, lie on his beds, and give him what he demanded.